src/api/authenticate/query/getAuthInfos.ts

```diff
@@ -0,0 +1,20 @@
+import { QueryFunctionContext, useQuery } from "@tanstack/react-query";
+import { authenticationKeys, checkApiKeyAgainstServer } from "..";
+
+const authTest = async ({
+  queryKey: [{ apiKey }],
+}: QueryFunctionContext<
+  ReturnType<(typeof authenticationKeys)["authentication"]>
+>) => checkApiKeyAgainstServer(apiKey);
+
+export const useAuthTest = ({
+  apikey,
+  enabled,
+}: { apikey?: string; enabled?: boolean } = {}) =>
+  useQuery({
+    queryKey: authenticationKeys.authentication(apikey),
+    queryFn: authTest,
+    enabled,
+    // TODO: use stale time to infinity?
+    // TODO: handle errors
```

Comment:
Are these still relevant, and if so, is there a DIR ticket for them?

Reply:
remove in 1f415da54e4faaf91a90bf5dbba3966e44797a52